docs/reference/upgrade/archived-settings.asciidoc

```diff
@@ -1,32 +1,29 @@
+[discrete]
 [[archived-settings]]
 == Archived settings
 
-If you upgrade a cluster with a deprecated persistent cluster setting to a
-version that no longer supports the setting, {es} automatically archives that
-setting. Similarly, if you upgrade a cluster that contains an index with an
+{es} typically removes support for deprecated settings at major version
+releases. If you upgrade a cluster with a deprecated persistent cluster setting
+to a version that no longer supports the setting, {es} automatically archives
+that setting. Similarly, if you upgrade a cluster that contains an index with an
```

Comment:
Drive by comment: I had to update this page recently as part of https://github.com/elastic/elasticsearch/pull/82069.

It looks like this reverts those changes. We should probably leave the page as it is now.